app/src/main/resources/db/migration/V1.0.0__Quarkus.sql

```diff
@@ -0,0 +1,151 @@
+DROP TABLE IF EXISTS CUSTOMER CASCADE;
```

Comment:
I would create issue to separate this into example application. 
We probably need example app with some of the non generic code pulled out from this. Generally this work end to end and it is very very good work, however it might need very small cleanup.